quinn-proto/src/connection.rs

```diff
@@ -515,7 +515,7 @@ impl Connection {
         self.space_mut(space).pending_acks.subtract(&info.acks);
     }
 
-    pub fn timeout(&mut self, now: Instant, timer: Timer) {
+    fn timeout(&mut self, now: Instant, timer: Timer) {
```

Comment:
I don't think there's any benefit to routing timers through the `Endpoint`, so keeping this public would allow a hypothetical decoupled configuration to e.g. run separate timer wheels for separate threads.

Reply:
Let's keep it private while it can be, and then we can always make it public again if a better way comes along?